packages/utils/src/compareStrings.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Given 2 strings, it determines if one is greater than the other or not.
```

Comment:
just asking: do we really need such documentation? it is not descriptive enough?

Reply:
Does it hurt @yauheni-deriv? 😂

Reply:
But I was just trying to get into the habit of commenting code, so this is practise. If it's intrusive, I can remove it 😁

Reply:
no man, it’s up to you :) 
or better ask other team mates for their opinion.
INHO, it’s better to apply proper naming and keep code as cleaner as we can and leave comment only in really not explicit places

Reply:
@jim-deriv It's a good thing, Let's keep it please 🙏🏻 

We can improve it tho, For example, we don't really need `@param` and `@returns` from JSDoc, We already have TS for those and it should be enough. But we can make the description more descriptive and meaningful.

- `compareStrings`, What kind of comparison? The function name could be more specific.
- `it determines if one is greater than the other or not`, What does it mean `greater`? The length? The value? Alphabetic order?

Reply:
@farzin-deriv, what would you suggest the name of the function should be?

Reply:
@jim-deriv To be honest I don't know what the function is doing yet 😅 Sorting alphabetically if I'm not mistaken, Right? 🤔

Reply:
Yes @farzin-deriv, sorting  "lexicographical"

Reply:
`LexicographicalStringSorter`? `AlphabeticallyStringSorter`? Something like that maybe? 🤔 @yauheni-deriv any recommendation?

By naming it properly then where we use it, It will also be readable;
```ts
['foo', 'bar', 'baz'].sort(AlphabeticallyStringSorter);
```

Reply:
yes, Alphabetically and considering this rules:
https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#import-rules